ta4j-examples/src/main/java/ta4jexamples/loaders/CsvBarsLoader.java

```diff
@@ -76,6 +76,11 @@ public static TimeSeries loadCsvSeries(String filename) {
         } catch (NumberFormatException nfe) {
             Logger.getLogger(CsvBarsLoader.class.getName()).log(Level.SEVERE, "Error while parsing value", nfe);
         }
+        try {
+            csvReader.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
Two try blocks with same catch clausel... Please use 
[try with ressource](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) block. Largely unknown, but best practices for closeable objects